.github/ISSUE_TEMPLATE/security_vulnerability.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Security vulnerability
+about: Create a report to help us improve our security
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Do not disclose details of the security issue**
+
+Please disclose you found a security vulnerability and admins will contact you
+to get details.
```

Comment:
Per Friday meeting discussion, let's not add a security vulnerability template and update the notice in the other templates to direct people to email us instead.